bundles/org.openhab.ui/web/src/pages/settings/things/inbox/inbox-list.vue

```diff
@@ -370,6 +366,9 @@ export default {
         console.error(err)
         this.$f7.dialog.alert('An error occurred: ' + err)
       })
+    },
+    filterSelectedItems () {
+      return this.inbox.filter((e) => this.selectedItems.indexOf(e.thingUID) >= 0)
```

Comment:
This would actually have been a textbook case for a computed property instead of a method ;)
Also just noticed there's a `isChecked` method so this could have been:
```js
computed: {
  ...
  selection () {
    return this.inbox.filter((e) => this.isChecked(e.thingUID))
  }
}
```
then `promises = this.selection.map(...)`

But enough with the hair splitting 😄 This "bulk selection/action" code (`selectedItem`, `toggleItemCheck` etc.)  is duplicated on several screens so there's a good chance it will be centralized in a mixin some day.